### DISCUSSION TITLE
USB HID problem

### DISCUSSION THREAD

**Discussion by oh3gdo** [community/community#136793] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey,  I have done PIC18F2550 USB serial data connection to PC and CCS c-complier v5.115. 
I got the hardware working 
http://probyte.fi/Dyno1.jpg

I added a  hid device to my table PC, 

Then I tried to find a HID operating system for serial bus. 
I found Zadig program, but it must to learn to my HID system.
I tries to do HID device from my list, the first in my list. 

Then all the mouse and keyboard stopped to to work. 
I have modified my keyboard and mouse (they go to my PC without wires)!
I looked a new keyboard and mouse to my PC and they started to work, but how I got old devices back? 

 Here is the help files
https://github.com/pbatard/libwdi/wiki/Zadig#user-content-The_Problem
It doesn't talk about going back.

[zadig](https://github.com/user-attachments/assets/f1bd7ed2-ef0f-487c-845c-7d84e036c4cc)
I loaded the first HID device from Device list, 
I was my mouse and keyboard! 
I looked the help files and then go to help discussion, 
http://janaxelson.com/forum/index.php?topic=1802.0  
bur it was already closed. 

I am in problem how to get the old HID back with this program or by  with a new program? 
I have tried to open and close my radio keyboard and mouse device. 
No help

Regards   
`personal identifiable information removed by moderator LiteBrite82`

**Comment by LiteBrite82** [comment]
Hi @og3gdo ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
